play.php

```diff
@@ -1,5 +1,5 @@
 <?php
 
 $game = include 'round.php';
-// $game = include 'round-2.php';
+$game = include 'round-2.php';
```

Comment:
Won't this override round one immediately?

Reply:
doesn't matter a ton, you have to do it manually at some point, but yeah I'll fix it